com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -65,6 +65,9 @@ private static string GetSceneEventTypeName(uint typeCode)
         private readonly EventMetric<ServerLogEvent> m_ServerLogReceivedEvent = new EventMetric<ServerLogEvent>(NetworkMetricTypes.ServerLogReceived.Id);
         private readonly EventMetric<SceneEventMetric> m_SceneEventSentEvent = new EventMetric<SceneEventMetric>(NetworkMetricTypes.SceneEventSent.Id);
         private readonly EventMetric<SceneEventMetric> m_SceneEventReceivedEvent = new EventMetric<SceneEventMetric>(NetworkMetricTypes.SceneEventReceived.Id);
+        private readonly Counter m_PacketSentCounter = new(NetworkMetricTypes.PacketSent.Id);
+        private readonly Counter m_PacketReceivedCounter = new(NetworkMetricTypes.PacketReceived.Id);
+
```

Comment:
Be careful with this type inference for `new`, it'll work in more recent versions of Unity, but not with the version of C# available in Unity < 2021.2 (IIRC)

Reply:
Thanks for the reminder will fix